test/sass/functions_test.rb

```diff
@@ -1520,18 +1546,45 @@ def test_call_uses_local_scope
   a: call(foo);
 }
 SCSS
+    end
+  end
+
+  def test_call_with_ref_uses_local_scope
```

Comment:
This was added when I was working on silencing deprecation warnings, I'll move it.